servercfg/serverconf.go

```diff
@@ -430,7 +430,13 @@ func GetPublicIP() (string, error) {
 	iplist := []string{"https://ip.server.gravitl.com", "https://ifconfig.me", "https://api.ipify.org", "https://ipinfo.io/ip"}
 	publicIpService := os.Getenv("PUBLIC_IP_SERVICE")
 	if publicIpService != "" {
-		logger.Log(3, "User provided public IP service is", publicIpService)
+		logger.Log(3, "User (environment variable) provided public IP service is", publicIpService)
+
+		// prepend the user-specified service so it's checked first
+		iplist = append([]string{publicIpService}, iplist...)
```

Comment:
This results in two appends.  Line 442 also appends

Reply:
Line 436 prepends for the environment variable case.  Line 442 is an else if and prepends for the config file case.

Reply:
gotcha, should have opened the entire file instead of just looking at the github diff.... spacing was off and thought 442 was outside the if block